easybuild/easyconfigs/p/PostgreSQL/PostgreSQL-9.6.2-foss-2016b-Python-2.7.12.eb

```diff
@@ -0,0 +1,46 @@
+easyblock = 'ConfigureMake'
+
+name = 'PostgreSQL'
+version = '9.6.2'
+versionsuffix = '-Python-%(pyver)s'
+
+homepage = 'https://www.postgresql.org/'
+description = """PostgreSQL is a powerful, open source object-relational database system.
+ It is fully ACID compliant, has full support for foreign keys,
+ joins, views, triggers, and stored procedures (in multiple languages).
+ It includes most SQL:2008 data types, including INTEGER,
+ NUMERIC, BOOLEAN, CHAR, VARCHAR, DATE, INTERVAL, and TIMESTAMP.
+ It also supports storage of binary large objects, including pictures,
+ sounds, or video. It has native programming interfaces for C/C++, Java,
+ .Net, Perl, Python, Ruby, Tcl, ODBC, among others, and exceptional documentation."""
```

Comment:
maybe, but I'm just using the same description as in the other `PostgreSQL` easyconfigs

If this is too long, we should change it consistently.